src/main/webapp/css/folder-strategy.css

```diff
@@ -62,19 +62,35 @@ div.role {
     background-color: firebrick;
 }
 
-button.submit-button {
+input[type="text"] {
+    margin-left: 3px;
+    border-radius: 5px;
+    border-style: solid;
+    border-width: thin;
+    padding: 3px 3px;
+}
+
+button.submit-button, input.submit {
     -webkit-border-radius: 5px;
     -moz-border-radius: 5px;
     border-radius: 5px;
     padding: 5px 5px;
-    margin-top: 10px;
+    margin-top: 5px;
     margin-left: 10px;
-    background-color: #dddddd;
-    display: block;
+    line-height: 1.25;
+    border-style: solid;
+    border-color: #007bff;
+    background-color: #007bff;
+    color: #ffffff;
```

Comment:
The buttons look like this now:

![image](https://user-images.githubusercontent.com/11471599/62354451-88ce8300-b52a-11e9-956d-8846cd87666c.png)